patches/server/0984-SculkCatalyst-bloom-API.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Oliwier Miodun <naczs@blueflow.pl>
+Date: Mon, 10 Jul 2023 17:59:42 +0200
+Subject: [PATCH] SculkCatalyst bloom API
+
+== AT ==
+public net.minecraft.world.level.block.entity.SculkCatalystBlockEntity$CatalystListener bloom(Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/core/BlockPos;Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/util/RandomSource;)V
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/block/CraftSculkCatalyst.java b/src/main/java/org/bukkit/craftbukkit/block/CraftSculkCatalyst.java
+index dc7d6c443af206f50d2b4ca260d64bb86b67553c..84d82dd7ae692b112d980a9e935dae96086ea0b1 100644
+--- a/src/main/java/org/bukkit/craftbukkit/block/CraftSculkCatalyst.java
++++ b/src/main/java/org/bukkit/craftbukkit/block/CraftSculkCatalyst.java
+@@ -9,4 +9,22 @@ public class CraftSculkCatalyst extends CraftBlockEntityState<SculkCatalystBlock
+     public CraftSculkCatalyst(World world, SculkCatalystBlockEntity tileEntity) {
+         super(world, tileEntity);
+     }
++
++    // Paper start - SculkCatalyst bloom API
++    @Override
++    public void bloom(org.bukkit.Location location, int charge) {
```

Comment:
In general, you can use the Position type instead of a location. This is because we don't want to have a world be a possible parameter, as it will always use the sculk's world anyway.